packages/scandipwa/src/component/CheckoutShipping/CheckoutShipping.container.js

```diff
@@ -46,18 +50,53 @@ export class CheckoutShippingContainer extends PureComponent {
         onShippingSuccess: this.onShippingSuccess.bind(this),
         onShippingError: this.onShippingError.bind(this),
         onAddressSelect: this.onAddressSelect.bind(this),
-        onShippingMethodSelect: this.onShippingMethodSelect.bind(this)
+        onShippingMethodSelect: this.onShippingMethodSelect.bind(this),
+        onStoreSelect: this.onStoreSelect.bind(this)
     };
 
     __construct(props) {
         super.__construct(props);
 
         const { shippingMethods } = props;
         const [selectedShippingMethod] = shippingMethods;
+        const { method_code = '' } = selectedShippingMethod || {};
 
         this.state = {
             selectedCustomerAddressId: 0,
-            selectedShippingMethod
+            selectedShippingMethod: method_code && method_code !== STORE_IN_PICK_UP_METHOD_CODE
+                ? selectedShippingMethod
+                : {}
+        };
+    }
+
+    getStoreAddress(shippingAddress) {
+        const {
+            selectedStoreAddress: {
+                region,
+                city,
+                postcode,
+                phone,
+                street,
+                name,
+                pickup_location_code
+            }
+        } = this.state;
+
+        return {
+            ...shippingAddress,
+            region,
+            city,
+            postcode,
+            telephone: phone,
+            street: [street],
+            firstname: name,
+            lastname: 'Store',
+            extension_attributes: [
+                {
+                    attribute_code: STORE_IN_PICK_UP_ATTRIBUTE_CODE,
+                    value: pickup_location_code
+                }
+            ]
```

Comment:
Liana says there is an option to disable this method. Should this really be there like... always?

Reply:
There is the check in method `onShippingSuccess` is it there or not:
![image](https://user-images.githubusercontent.com/43142475/112822550-c46b7880-9090-11eb-8a4a-d7068ca47929.png)